lib/src/summary.dart

```diff
@@ -107,6 +105,9 @@ class Summary extends Object with _$SummarySerializerMixin {
   @JsonKey(includeIfNull: false)
   final List<Suggestion> suggestions;
 
+  final PkgResolution pkgResolution;
```

Comment:
can we do this in a separate change? this is going to be big with all the test data changes...

Reply:
ah poo...forgot about tests. Yup

Reply:
PTAL